### DISCUSSION TITLE
storage.py: Local import of localforage.js instead of a CDN

### DISCUSSION THREAD

**Discussion by benlawraus** [anvilistas/anvil-extras#403] [Q&A]
Hi,
In your storage.py:
```
try:
    _forage = _window.localforage
except AttributeError:
    _ForageModule = anvil.js.import_from(
        "https://cdn.skypack.dev/pin/localforage@v1.10.0-vSTz1U7CF0tUryZh6xTs/mode=imports,min/optimized/localforage.js"
    )
    _forage = _ForageModule.default

```
can you help me rewrite this so that I can have the localforage module as an asset?
The following code is wrong, because  _ForageModule doesn't have a default.
```
_ForageModule = anvil.js.import_from("./_/theme/localforage.js")
_forage = _ForageModule.default
```

Thank you for your time!

**Comment by s-cork** [comment]
Add a script tag in your native libraries in the standard way where the src is `"_/theme/..."`

This will put localForage on window and will stop anvil-extras importing it from CDN.
